docs/content/http-api-authorization.md

```diff
@@ -262,7 +262,7 @@ default allow := false
 allow {
     some username
     input.method == "GET"
-    input.path = ["finance", "salary", username]
+    input.path == ["finance", "salary", username]
     token.payload.user == username
     user_owns_token
```

Comment:
I understand that unification is confusing and rare. Let's write this without then:

```rego
allow {
    input.method == "GET"
    input.path == ["finance", "salary", token.payload.user]
    user_owns_token
}
```

or 
```rego
allow {
    username := token.payload.user
    input.method == "GET"
    input.path == ["finance", "salary", username]
    user_owns_token
}
```

Reply:
Yes, it makes sense to me, thank you for clarification!